<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>367</width>
    <height>555</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>367</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Send To LAN</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="LoadFileButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StartButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CB_devList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SetNetwork_PB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Network</string>
     </property>
    </widget>
    <widget class="QLabel" name="GapTime_Lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gap Time (ms)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="GapTimeVal_LE">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="BitRate_Lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bit Rate :</string>
     </property>
    </widget>
    <widget class="QLabel" name="BitRateVal_Lbl">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="FrameLen_Lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame Length</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FrameLenVal_LE">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1024</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadSize_Lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ReadSizeVal_LE">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>20480</string>
     </property>
    </widget>
    <widget class="QLabel" name="PortNum_Lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PortNum_LE">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1234</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadSize_Lbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Repeat:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NumRep_LE">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
